lib/mongo/auth/kerberos.rb

```diff
@@ -28,6 +28,18 @@ class Kerberos
       # @since 2.0.0
       MECHANISM = 'GSSAPI'.freeze
 
+      # Instantiate a new authenticator.
+      #
+      # example Create the authenticator.
+      #   Mongo::Auth::Kerberos.new(user)
+      #
+      # @param [ Mongo::Auth::User ] user The user to authenticate.
+      #
+      # @since 2.0.1
+      def initialize(user)
```

Comment:
`user` parameter being required is an API change, if this class is part of the public API. At the same time it is unclear to me whether version 2.0.0 worked at all, if it didn't and this is an API change I would make it into 2.1.0.

Reply:
The old version did not work at all; it's supposed to be duck-typed to match the other Authenticator classes in the driver (which all take a user as part of the constructor). See the below `login` method, where `user` was used without it being defined anywhere.

I'm fine with changing this to 2.1.0; I'll make the change.